release/rllib_tests/smoke_tests/smoke_test_basic_multi_node_training_learner.py

```diff
@@ -5,7 +5,6 @@
 
 def run_with_tuner_n_rollout_worker_2_gpu(config):
     """Run training with n rollout workers and 2 learner workers with gpu."""
-    config = config.rollouts(num_rollout_workers=5)
```

Comment:
I'd expect the test to run with the number of rollout workers given be the config, not 5.